### ISSUE TITLE
pyd files

### ISSUE THREAD

**Issue by Hanemann** [bleachbit/bleachbit#355]
Hello!

I uninstalled Beachbit in order to install version 2. But as soon as I  tried to open the set up exe, I got a number of error messages that various pyd files could not be found, so I had to abort. I d/l a new exe, but again the same problem.

How to solve this problem? Do I need all those pyd files?

**Comment by az0** [comment]
Would you please take a screenshot of the error and post it here? The
screenshot will help me figure out the problem

GitHub allows images as attachments, or you can post a link to the image
hosted elsewhere.In case it is useful, the Snag.gy site lets you both take
a screenshot and host it: https://snag.gy/

**Comment by Hanemann** [comment]
Now this is unbelievable.... After I had tried at least three times to install Bleachbit and every time I got the same error messages and had deleted all the exe's to the recycle bin, I restored one set up exe and now it worked!!! I have no idea what the problem was.

I can give you the first three error messages which were about the missing files as I had noted them: _ctypes.pyd; _hashlib.pyd and _socket.pyd. Every time the same messages and then I just aborted. 

Since the installation now worked, I cannot make screen shots. But perhaps it is of interest for you just to know what the error messages were. 

In any case, thank you for trying to help! Perhaps you have a magic touch, otherwise I cannot understand how all of a sudden the installation worked!
